os/include/tinyara/lwnl/lwnl80211.h

```diff
@@ -0,0 +1,264 @@
+/****************************************************************************
+ *
+ * Copyright 2019 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+#ifndef __INCLUDE_TINYARA_LWNL80211_H__
+#define __INCLUDE_TINYARA_LWNL80211_H__
+
+#include <tinyara/config.h>
+#include <tinyara/fs/ioctl.h>
+
+#include <stdint.h>
+#include <stdbool.h>
+#include <mqueue.h>
+#include <semaphore.h>
+
+/****************************************************************************
+ * Pre-processor Definitions
+ ****************************************************************************/
+#define LWNL80211_PATH "/dev/lwnl80211"
+
+#ifdef CONFIG_LWNL80211_DEBUG
+#define LWNL80211_LOG printf
+#else
+#define LWNL80211_LOG ndbg
+#endif
```

Comment:
This logic looks much strange.
Our dbg supports selection of logging level and module. And we can save it in RAM through ram syslog.
But this debug feature is different. I don't understand why only this module works differently,
When I disable LWLN80211_DEBUG and enable NETWORK_DEBUG, I can see lwnl's logs.
It does not make sense.

Reply:
We decide not to use ndbg logging. Thanks for the comment.